aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1 +1,361 @@
+- What does "out of memory" actually mean? We should be more specific.
+
+**Image goes here**
+
+Problems:
+- Strongly consider hyperlinking any named resource so that a user can directly navigate to it from a notification.
+- 95% of 10TB means that 500GB are still free. But, 95% of 16GB means only 800MB remain. Showing only percentages doesn't offer the user enough context to make a decision.
+
+**Image goes here**
+
+Problems:
+- What is the user supposed to do about this? Never raise an issue without offering a solution.
+
+**Image goes here**
+
+Problems:
+- What is the needful? This is a saying not used in conversational American English and does not give the user specific guidance.
+
+**Image goes here**
+
+Problems:
+- The user purchased the cluster to use it. Telling them to stop using it is not going to go over well.
+- Is this really a critical issue? Unused resources are often considered wasted resources. It is actually a good thing to be fully utilizing your hardware.
+- In an aaS world, additional capacity should be a simple add-on to an existing service. As such, this may not be a critical issue, but could be transformed to a lower priority marketing message.
+
+**Image goes here**
+
+Problems:
+- If possible, give the user a way to directly perform a mentioned action.
+
+**Image goes here**
+
+## Miscellany
+
+### Quoting
+
+In messaging that contains user-specified content, use double-quotes around this content if it is not rendered as a hyperlink. If user-content is rendered as a hyperlink, the hyperlink styling is sufficient to offset the content from other text in the message.
+
+Why is this important? User-entered names can contain spaces, and if they are not quoted or rendered as hyperlinks, they can hinder comprehension of a message.  
+
+Here is a correct example: *the uplink set "{0}" has a problem*. Compare this to an incorrect example: *the uplink set {0} has a problem*. If the uplink set is named "deleted", this will read as *the uplink set deleted has a problem*.
+
+### Referring to support
+
+- Do not tell the user to contact their administrator. The user is likely the administrator.
+- Do not tell the user to contact HPE. A 3rd party may be providing support to the customer.
+- Use something like: *Contact your authorized support representative.*
+- Do not overuse this phrase. If the resolution to a problem is clear, just give the resolution and do not add that the user should contact support.
+- Contacting support costs everypone time and money, so do not turn every issue into a potential support issue. It makes our product seem unnecessarily fragile and can result in unnecessary support calls.
+- Do use this phrase for internal errors where there's nothing else the user can do (i.e. when it is the only resolution).
+
+### Localization
```

Comment:
We've recently added a page with Internationalization/Localization guidance. Should we link it here? https://design-system.hpe.design/templates/internationalization